html/changelogs/changelog20182.html

```diff
@@ -1041,7 +1041,7 @@ <h3 class='author'>fosstar updated:</h3>
 		</ul>
 		<h3 class='author'>Suicidalpickles updated:</h3>
 		<ul class='changes bgimages16'>
-			<li class='imageadd'>Replaced zombie sprites with /tg/Station counterparts.</li>
+			<li class='imageadd'>Replaced zombies sprites with /tg/Station counterparts.</li>
```

Comment:
```suggestion
			<li class='imageadd'>Replaced zombie sprites with /tg/Station counterparts.</li>
```
Don't try to screw with the changelog file...